api/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/containerd/containerd/api
+
+go 1.16
+
+require (
+	github.com/containerd/ttrpc v1.0.2
+	github.com/containerd/typeurl v1.0.2
+	github.com/gogo/googleapis v1.4.1
+	github.com/gogo/protobuf v1.3.2
+	github.com/opencontainers/go-digest v1.0.0
+	google.golang.org/grpc v1.38.0
```

Comment:
Do we want / must these versions to always be in sync with the main `go.mod` ?

If so, perhaps we can script that, to make sure that they're updated whenever we update one of these in the main `go.mod`

Otherwise, there's a risk we run tests against the version in `vendor`, and not the versions specified here

Reply:
So for now adding the tip in root go.mod to update this file as well. +1 to add a script as follow up.